tests/libtest.sh

```diff
@@ -30,6 +30,10 @@ else
   test_builddir=$(dirname $0)
 fi
 
+if [ -z "${FUSERMOUNT}" ]; then
+  FUSERMOUNT=fusermount
+fi
```

Comment:
Probably better to do the equivalent of https://github.com/flatpak/flatpak/pull/5751